src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -285,6 +291,16 @@ private static void createBranches(List<LfBus> lfBuses, LfNetwork lfNetwork, Loa
             addBranch(lfNetwork, LfDanglingLineBranch.create(danglingLine, lfNetwork, lfBus1, lfBus2), report);
         }
 
+        for (TieLine tieLine : loadingContext.tieLines) {
+            LfTieLineBus lfBoundaryBus = new LfTieLineBus(lfNetwork, tieLine);
+            lfNetwork.addBus(lfBoundaryBus);
+            lfBuses.add(lfBoundaryBus);
+            LfBus lfBus1 = getLfBus(tieLine.getTerminal1(), lfNetwork, parameters.isBreakers());
+            LfBus lfBus2 = getLfBus(tieLine.getTerminal2(), lfNetwork, parameters.isBreakers());
+            addBranch(lfNetwork, LfTieLineBranch.create(tieLine, Branch.Side.ONE, lfNetwork, lfBus1, lfBoundaryBus), report);
+            addBranch(lfNetwork, LfTieLineBranch.create(tieLine, Branch.Side.TWO, lfNetwork, lfBus2, lfBoundaryBus), report);
```

Comment:
This is not correct `addBranch(lfNetwork, LfTieLineBranch.create(tieLine, Branch.Side.TWO, lfNetwork, lfBus2, lfBoundaryBus), report);` because of this scheme and your conversion, even if most of the time the half line is symmerical:
![image](https://user-images.githubusercontent.com/48123713/133581966-d078ce3d-8848-43c4-a189-8ed44b3faa0b.png)
For me, you should do `addBranch(lfNetwork, LfTieLineBranch.create(tieLine, Branch.Side.TWO, lfNetwork, lfBoundaryBus, lfBus2), report);`